public/app/features/org/NewOrgPage.tsx

```diff
@@ -41,12 +41,12 @@ export const NewOrgPage: FC<PropsWithState> = ({ navModel }) => {
   return (
     <Page navModel={navModel}>
       <Page.Contents>
-        <h3 className="page-sub-heading">New Organization</h3>
+        <h3 className="page-sub-heading">New organization</h3>
 
         <p className="playlist-description">
-          Each organization contains their own dashboards, data sources and configuration, and cannot be shared between
-          orgs. While users may belong to more than one, multiple organization are most frequently used in multi-tenant
-          deployments.{' '}
+          Each organization contains their own dashboards, data sources, and configuration, which cannot be shared
+          organizations. While users might belong to more than one organization, multiple organizations are most
```

Comment:
Is the "between" removed on purpose? This reads a bit weird to me: 
> which cannot be shared organizations

Reply:
Good catch. That was my goof.